### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by oiskritsky**
done all task.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+let user = "John Doe";
+let student;
+
+console.log(user); //"John Doe"
+console.log(student); //"Undefined"
+
+let student = "Oleg";
```
Comment (ghost) [inline js/main.js]: `student` was defined on line 2, remove `let` operator.

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		++result; //на 0 умножать нельзя
+	} else {
+		result *= arr[i];
+	}
+}
+
+console.log(result);
+
+for (let i = 0; arr.length > i; i++) {
+	if (arr[i] > 5 && arr[i] < 10) {
```
Comment (ghost) [inline js/main.js]:
```suggestion
5 < arr[i] < 10
```

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+		console.log(arr[i]);
+	}
+}
+
+for (let i = 0; arr.length > i; i++) {
+	if (arr[i] % 2 === 0 && arr[i] != 0) {
```
Comment (ghost) [inline js/main.js]:
```suggestion
  !(arr[i] % 2)
```
`0` also is even.

File: js/main.js (hunk 2 above)
Comment (oiskritsky) [reply js/main.js]: 5 < arr[i] < 10 its does't worked. "for" returns all available numbers in "arr".

File: js/main.js (hunk 1 above)
Comment (oiskritsky) [reply js/main.js]: I fixed it.

File: js/main.js (hunk 3 above)
Comment (oiskritsky) [reply js/main.js]: I fixed it.

File: js/main.js (hunk 2 above)
Comment (ghost) [reply js/main.js]: you are right.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
h

File: js/main.js (hunk 2 above)
Comment (BubentsovKirill) [reply js/main.js]: correct

**Pull request merged**